CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ endif()
 
 project(
     casbin
-    VERSION 1.38.0
+    VERSION 1.43.0
```

Comment:
```suggestion
    VERSION 1.43.1
```

You need to look ahead of time for the version to coincide with the release version bundled by GH. Since this PR is under the `chore` category, only the most minor version number will increment.

Reply:
We don't define release rule, so maybe we use the [default rule](https://github.com/semantic-release/commit-analyzer/blob/master/lib/default-release-rules.js). In defult rule, `chore` is [not defined](https://github.com/semantic-release/commit-analyzer/blob/master/README.md) and not related to a release.
![image](https://user-images.githubusercontent.com/43725202/143517782-92df384e-69ff-4023-9770-775c810c80eb.png)

Reply:
Maybe we can ref commit in [this chore](https://github.com/casbin/casbin-cpp/commit/0c42d80aa6ef87f6f8bad7979e23e418df23208c). And not create a release.
![image](https://user-images.githubusercontent.com/43725202/143518378-a6ac341c-3e6d-4cd8-987d-f81842bb00da.png)

![image](https://user-images.githubusercontent.com/43725202/143518364-ec2b2a11-d163-444d-99cf-cb309fb022b6.png)

Reply:
Yes, that's right. Good observation @sheny1xuan!